src/custom-hooks/useOktaTokens.ts

```diff
@@ -0,0 +1,42 @@
+const getField = (storageKey, field) => {
```

Comment:
can we not use this from madie-util?

Reply:
Ah, this file can be deleted. It's leftover from my original attempt at autologout.